src/components/application_manager/include/application_manager/application_data_impl.h

```diff
@@ -45,6 +45,8 @@ namespace application_manager {
 
 namespace mobile_api = mobile_apis;
 
+typedef std::map<uint32_t, bool> ChoiceSetAllowedMap;
```

Comment:
@ychernysheva I think it's simpler to have just a `set<uint32_t>` in this case. For `set_choice_set_allow_mode(true)` just add a choice set to collection, for `set_choice_set_allow_mode(false)` remove it from collection, for `is_choice_set_allowed` it will be true if collection contains this choice, false otherwise